### PR TITLE
`Get Help` modal for extensions 

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR adds the `get help` beside troubleshoot guide which was added in extensions but not through meshery

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/TroubleshootingComponent.js [hunk 1]
```diff
@@ -1,24 +1,86 @@
...
+    setOpenForm(true);
+  };
+
+  const handleSupportFormClose = () => {
+    setOpenForm(false);
+  };
+
+  const handleSupportFormSubmission = async (data) => {
+    try {
+      await axios.post('https://hook.us1.make.com/r5qgpjel5tlhtyndcgjvkrdkoc65417y', {
```
Comment (leecalcote) [inline ui/components/TroubleshootingComponent.js:32]: @sudhanshutech, thanks! Please note that this webhook cannot be included in Meshery directly, but must be in an extension.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/TroubleshootingComponent.js (hunk 1 above)
Comment (MUzairS15) [reply ui/components/TroubleshootingComponent.js:32]: @yash37158 Will you add an API in meshery server that proxies this support request to remote provider and then the provider (in the backend) handles this?

Comment (yash37158) [reply ui/components/TroubleshootingComponent.js:32]: @MUzairS15 Yeah sure

Comment (leecalcote) [reply ui/components/TroubleshootingComponent.js:32]: @Yashsharma1911 ^^

Comment (leecalcote) [reply ui/components/TroubleshootingComponent.js:32]: @sudhanshutech ?

**Comment by sudhanshutech** [comment]
@Yashsharma1911 
image

File: ui/components/TroubleshootingComponent.js [hunk 2]
```diff
@@ -1,24 +1,86 @@
...
+    setOpenForm(false);
+  };
+
+  const handleSupportFormSubmission = async (data) => {
+    try {
+      await axios.post('api/extensions/api/webhook/support', {
+        memberFormOne: {
+          ...data,
+          name: userData?.first_name + ' ' + userData?.last_name,
+          email: userData?.email,
```
Comment (Yashsharma1911) [inline ui/components/TroubleshootingComponent.js:36]: I think this is not correct payload right @sudhanshutech ? validation will be failing

Comment (Yashsharma1911) [reply ui/components/TroubleshootingComponent.js:36]: Please make sure we are passing same payload that we were doing in cloud to make sure validation get's passed

Comment (sudhanshutech) [reply ui/components/TroubleshootingComponent.js]: its fixed and payload is now correct
image
image

**Comment by leecalcote** [comment]
@sudhanshutech merge conflict. 

Please merge PR once conflict is addressed.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**